hi-ml-histopathology/environment.yml

```diff
@@ -15,13 +15,14 @@ dependencies:
       - conda-merge==0.1.5
       - cryptography==3.3.2
       - docker==4.3.1
+      - flask==2.0.1
       - gitpython==3.1.7
       - gputil==1.4.0
       - hi-ml>=0.1.12
       - joblib==0.16.0
       - jupyter==1.0.0
       - jupyter-client==6.1.5
-      - lightning-bolts==0.5.0
+      - lightning-bolts==0.4.0
```

Comment:
for my own knowledge why degrading is needed?

Reply:
I upgraded it by accident in the last PR and it was out of scope of these 2 PRs to make all the changes and make sure the new version wouldn't break anything